pjsip/include/pjsua-lib/pjsua.h

```diff
@@ -4921,6 +4934,25 @@ PJ_DECL(pj_status_t) pjsua_acc_get_config(pjsua_acc_id acc_id,
 PJ_DECL(pj_status_t) pjsua_acc_modify(pjsua_acc_id acc_id,
                                       const pjsua_acc_config *acc_cfg);
 
+/**
+ * Send arbitrary requests using the account. Application should only use
+ * this function to create auxiliary requests outside dialog, such as
+ * OPTIONS, and use the call or presence API to create dialog related
+ * requests.
+ *
+ * @param acc_id        The account ID.
+ * @param dst_uri       URI to be put in the To header (normally is the same
+ *                      as the target URI).
+ * @param method        The SIP method of the request.
+ * @param msg_data      Optional headers etc to be added to outgoing
+ *                      request, or NULL if no custom header is desired.
+ *
+ * @return              PJ_SUCCESS or the error code.
+ */
+PJ_DECL(pj_status_t) pjsua_acc_send_request(pjsua_acc_id acc_id,
+                                            const pj_str_t *dest_uri,
+                                            const pj_str_t *method_str,
+                                            const pjsua_msg_data *msg_data);
```

Comment:
Add few params which perhaps can be useful in the future:
- app user-data/token (void *),
- generic options/settings (void *): for now perhaps must be NULL, for extensibility in the future, so we don't need to create `pjsua_acc_send_request2()` :)

Reply:
Looks like only token/user-data was added and it is specified as "unused" while it should be usable (as opposed to must-be-NULL :)

Actually user-data/token and settings are actually different:
- user-data/token, basically it needs to be passed back in the callback, so app can correspond the tsx to something, just check `pjsip_endpt_send_request()` or `pjsua_call_make_call()` for reference.
- settings, it is for future use, must be NULL for now.

Reply:
Sorry..I misunderstood.

So you are saying that the token should already work and shouldn't be for future use cases, i e. that an object is allocated in the pool, which stores the acc_id and the token (a pointer). (In current PR only acc_id is stored).